pom.xml

```diff
@@ -575,6 +575,11 @@
                 <artifactId>netty-transport</artifactId>
                 <version>${netty.version}</version>
             </dependency>
+            <dependency>
```

Comment:
is this new jar getting bundled ? please check

Reply:
Added required changes to pom.xml files. Please review.